scalableminds/webknossos
# Fix empty tree list and some tree list errors

- Providing the width and height from the AutoSizer component explicitly as `reactVirtualizedListProps` to the `SortableTree` component solved the empty tree list for me. Seems like the `SortableTree` tried to figure those out itself otherwise which failed often. The library updates didn't help, but I kept them anyways.
- I added two more errors from the `SortableTree`'s drag-and-drop library to the ignored error list as they don't seem to have any serious effect but happen from time to time while drag-and-dropping :man_shrugging: 
- I also fixed an error when multi-selecting trees and multi-selecting a group afterwards. (part of #4789)

### URL of deployed dev instance (used for testing):
- https://fixhiddentreelist.webknossos.xyz

### Steps to test:
- Open a tracing where the tree list is visible and reload a couple of times. The tree list should always be visible.
- Multi-select some trees, then multi-select a group and click "Ok" -> There shouldn't be an error.

### Issues:
- fixes #4807 
- contributes to #4789

------
